frontstage_api/controllers/case_controller.py

```diff
@@ -19,8 +19,7 @@ def get_case_by_case_id(case_id):
     response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
 
     if response.status_code != 200:
-        logger.error('Failed to retrieve case', case_id=case_id)
-        raise ApiError(url, response.status_code)
+        raise ApiError(url, response.status_code, description='Failed to retrieve case', case_id=case_id)
```

Comment:
Shouldn't this be url=url, statuscode=response.statuscode?

Reply:
I don't think the kwargs need to be formally assigned as it is pretty self explanatory variable names but I do think it needs to be consistent

Reply:
I know it's been done like my comment in Secure Messaging

Reply:
It has which is fine. Should be consistent in the services and across the services but it doesn't strictly need it. As this would need changed in a lot of other places to